core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/stopstartus/StopStartUpscaleActions.java

```diff
@@ -137,17 +154,16 @@ protected void doExecute(StopStartUpscaleContext context, StopStartUpscaleStartI
                 List<InstanceMetadataView> allInstanceMetadata = instanceMetaDataService.getAllAvailableInstanceMetadataViewsByStackId(stack.getId());
                 List<InstanceMetadataView> startedInstancesMetaData = cloudInstanceIdToInstanceMetaDataConverter.getNotDeletedAndNotZombieInstances(
                         allInstanceMetadata, context.getHostGroupName(), cloudInstanceIdsStarted);
-                clusterUpscaleFlowService.instancesStarted(stack.getId(), startedInstancesMetaData);
-
-                handleInstanceUnsuccessfulStart(context, cloudVmInstanceStatusList);
 
-                // This list is currently empty. It could be populated later in another flow-step by querying CM to get service health.
-                // Meant to be a mechanism which detects cloud instances which are RUNNING, but not being utilized (likely due to previous failures)
                 List<CloudInstance> instancesWithServicesNotRunning = payload.getStartInstanceRequest().getStartedInstancesWithServicesNotRunning();
                 List<InstanceMetadataView> metaDataWithServicesNotRunning = cloudInstanceIdToInstanceMetaDataConverter.getNotDeletedAndNotZombieInstances(
                         allInstanceMetadata,
                         context.getHostGroupName(),
-                        instancesWithServicesNotRunning.stream().map(i -> i.getInstanceId()).collect(Collectors.toUnmodifiableSet()));
+                        instancesWithServicesNotRunning.stream().map(CloudInstance::getInstanceId).collect(Collectors.toUnmodifiableSet()));
+
+                clusterUpscaleFlowService.instancesStarted(stack.getId(), startedInstancesMetaData);
```

Comment:
The logging here needs to differentiate which instances have artificially been marked as STARTED (recovery candidates) vs which instances were actually started by invoking a cloud-provider start API. See note in StopStartUpscaleStartInstancesHandler

Reply:
Updated the message propagated to the activity console